### DISCUSSION TITLE
[PSD6] Example 4

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#144] [Esercizi (A-L)]
Semplificare la seguente equazione booleana:

![image](https://user-images.githubusercontent.com/69389461/210016505-4b7c0c63-bbd5-4367-bf78-277b201e1142.png)

**Comment by Elia-Belli** [comment]
$A\bar B C + ABC + \bar A B C$
$AC(\bar B + B) + BC (A + \bar A)$ (duplicando ABC)
$AC + BC$

**Comment by FeddyLix17** [comment]
corretto
